quay-replication-cr.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: harbor-configuration.harbor.configuration/v1alpha1
+kind: HarborConfiguration
```

Comment:
Same as above, should be in `samples` subfolder I guess.

Reply:
It has been moved to the samples folder